src/server/repository/assessmentCycle/data/getFraYearsDataQuery.ts

```diff
@@ -0,0 +1,765 @@
+
+       gos.masters_degree_total                                     as "7b_msc_tot",
+       gos.masters_degree_female                                    as "7b_msc_fem",
+       gos.masters_degree_male                                      as "7b_msc_male",
+
+       gos.bachelors_degree_total                                   as "7b_ba_tot",
+       gos.bachelors_degree_female                                  as "7b_ba_fem",
+       gos.bachelors_degree_male                                    as "7b_ba_male",
+
+       gos.technician_certificate_total                             as "7b_tec_tot",
+       gos.technician_certificate_female                            as "7b_tec_fem",
+       gos.technician_certificate_male                              as "7b_tec_male",
+
+       gos.total_total                                              as "7b_total_tot",
+       gos.total_female                                             as "7b_total_fem",
+       gos.total_male                                               as "7b_total_male"
+
+from ${schemaCycle}.country cc
+         join _regions r using (country_iso)
+         join _years y using (country_iso)
+         join climaticdomain c using (country_iso)
+
+--       1
+         left join extentofforest eof using (country_iso, year)
+         left join forestcharacteristics fc using (country_iso, year)
+         left join specificforestcategories sfc using (country_iso, year)
+         left join otherlandwithtreecover olwtc using (country_iso, year)
+--       2
+         left join growingstockavg gsavg using (country_iso, year)
+         left join growingstocktotal gstotal using (country_iso, year)
+         left join growingstockcomposition gscomposition using (country_iso, year)
+
+         left join biomassstock bms using (country_iso, year)
+
+         left join carbonstock cs using (country_iso, year)
+
+--       3
+         left join primarydesignatedmanagementobjective pdmo using (country_iso, year)
+         left join totalareawithdesignatedmanagementobjective tawdmo using (country_iso, year)
+
+         left join forestAreaWithinProtectedAreas fawpa using (country_iso, year)
+--       4
+         left join forestOwnership fo using (country_iso, year)
+         left join holderofmanagementrights homr using (country_iso, year)
+--        6
+         left join areaofpermanentforestestate aopfe using (country_iso, year)
+--        7
+         left join employment e using (country_iso, year)
+         left join graduationofstudents gos using (country_iso, year)
+    -- special cases
+-- 2 no year
+         left join carbonstocksoildepth cssd on (cc.country_iso = cssd.country_iso) -- no year, single col
+-- 5 no year
+         left join degradedforest df on (cc.country_iso = df.country_iso) -- no year
+-- 6 no year
+         left join forestpolicy fp on (cc.country_iso = fp.country_iso) -- no year
+
+where cc.country_iso not ilike 'X%'
```

Comment:
this query is very difficult to maintain and understand.
no comments, no dynamic generations. this cannot be committed like this. 
a more depth review is needed.